PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -171,3 +171,11 @@
 _phase2_timing_extraCommands = ["keep *_offlineSlimmedPrimaryVertices4D_*_*"]
 from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
 phase2_timing.toModify(MicroEventContentMC, outputCommands = MicroEventContentMC.outputCommands + _phase2_timing_extraCommands)
+
+_pp_on_AA_extraCommands = [
+    'keep booledmValueMap_*MuonID_*_*',
```

Comment:
wildcards in module names should not be used; the goal is to select everything that's needed explicitly.
'keep booledmValueMap_packedPFCandidateMuonID_*_*'

Reply:
Removed the wildcard on the module name